fe1-web/src/App.tsx

```diff
@@ -29,6 +30,42 @@ configureNetwork(messageRegistry, keyPairRegistry);
 // start persisting the redux state after all reducers have been registered
 persist.persist();
 
+const BuildInfo = () => {
+  const styles = StyleSheet.create({
+    container: {
+      position: 'absolute',
+      bottom: '4px',
+      left: '3px',
+      zIndex: 100,
+      color: '#757575',
```

Comment:
If possible colors from the color scheme (https://github.com/dedis/popstellar/blob/master/fe1-web/src/core/styles/color.ts) are preferred so that they can be changed all at once. `inactive` (not the best name for sure) / #8E8E8E should be similar to #757575.